apps/src/templates/UnitSelectorV2.jsx

```diff
@@ -6,6 +6,22 @@ import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
 import {loadUnitProgress} from '@cdo/apps/templates/sectionProgress/sectionProgressLoader';
 import {setScriptId} from '@cdo/apps/redux/unitSelectionRedux';
+import firehoseClient from '../lib/util/firehose';
+
+const recordEvent = (eventName, sectionId, dataJson = {}) => {
```

Comment:
No clue how this didn't get caught when testing the unit selector, but it was throwing a console error.